src/search.ts

```diff
@@ -68,7 +68,8 @@ async function* indexPages(config: Config, effects: SearchIndexEffects): AsyncIt
   }
 
   for await (const path of config.paths()) {
```

Comment:
TODO It would be better to use the list of paths already computed during build rather than calling `config.paths` again.